aldryn_jobs/models.py

```diff
@@ -326,6 +327,7 @@ class Meta:
 class JobListPlugin(BaseJobsPlugin):
     """ Store job list for JobListPlugin. """
     jobopenings = SortedManyToManyField(JobOpening, blank=True, null=True,
+        verbose_name=_('job openings'),
         help_text=_("Select Job Openings to show or don't select any to show "
                     "last job openings. Note that Job Openings form different "
```

Comment:
Suggest: "Select Job Openings to show. If none are selected, the most recent Job Openings from the selected app-config will be shown."